From 61388da37e9eacfe107ce1139707250c21e4d28b Mon Sep 17 00:00:00 2001
From: Hien Dang <hien.dang.eb@rvc.renesas.com>
Date: Fri, 18 Mar 2016 14:19:20 +0700
Subject: [PATCH 1291/2066] thermal: renesas: Add compatible string for
 r8a7796

New thermal compatible string for r8a7796 is added.
Also remove "renesas,rcar-gen3-thermal" because it is
of no use now.

Signed-off-by: Hien Dang <hien.dang.eb@rvc.renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 .../bindings/thermal/rcar-gen3-thermal.txt         |    8 +++-----
 1 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/Documentation/devicetree/bindings/thermal/rcar-gen3-thermal.txt b/Documentation/devicetree/bindings/thermal/rcar-gen3-thermal.txt
index f5c9430..e87123c 100644
--- a/Documentation/devicetree/bindings/thermal/rcar-gen3-thermal.txt
+++ b/Documentation/devicetree/bindings/thermal/rcar-gen3-thermal.txt
@@ -2,9 +2,9 @@
 
 Required properties:
 - compatible		: "renesas,thermal-<soctype>",
-			   "renesas,rcar-gen3-thermal" (with thermal-zone always)
 			  Examples with soctypes are:
 			    - "renesas,thermal-r8a7795" (R-Car H3)
+			    - "renesas,thermal-r8a7796" (R-Car M3)
 - reg			: Address range of the thermal registers.
 - #thermal-sensor-cells : Please see ./thermal.txt
 
@@ -15,8 +15,7 @@ Option properties:
 Example (non interrupt support):
 
 thermal: thermal@0xe6198000 {
-                        compatible = "renesas,thermal-r8a7795",
-                                "renesas,rcar-gen3-thermal",
+                        compatible = "renesas,thermal-r8a7795";
                         reg = <0 0xe6198000 0 0x5c>;
                 };
 
@@ -32,8 +31,7 @@ thermal-zones {
 Example (interrupt support):
 
 thermal: thermal@0xe6198000 {
-                        compatible = "renesas,thermal-r8a7795",
-                                "renesas,rcar-gen3-thermal",
+                        compatible = "renesas,thermal-r8a7795";
                         reg = <0 0xe6198000 0 0x5c>;
 			interrupts = <GIC_SPI 67 IRQ_TYPE_LEVEL_HIGH>;
                 };
-- 
1.7.5.4

